suites/reef/rados/tier-2_rados_test-bugfixes.yaml

```diff
@@ -150,33 +150,51 @@ tests:
           plugin: jerasure
           disable_pg_autoscale: true
         inconsistent_obj_count: 4
+        replicated_pool:
+          create: true
+          pool_name: replicated_pool
+          pg_num: 1
         delete_pool:
           - ecpool
-
+  - test:
```

Comment:
Add this bugzilla details in the comment: https://bugzilla.redhat.com/show_bug.cgi?id=2299470, so that we can see the active bug in the test report upon failure